tests/introspection/tests.py

```diff
@@ -211,3 +211,132 @@ def test_get_constraints_indexes_orders(self):
                 self.assertEqual(val['orders'], ['ASC'] * len(val['columns']))
                 indexes_verified += 1
         self.assertEqual(indexes_verified, 4)
+
+    def test_get_constraints(self):
+        def assert_details(first, second):
+            # different backend introspection realizations has different data
+            first = {
+                k: v for k, v in first.items()
+                if k in {'columns', 'index', 'unique', 'primary_key', 'check', 'foreign_key'}
+            }
+            # different backend introspection realizations has different values for same constraints
```

Comment:
added difference, but I really don't know how to describe why, because it's look like no same behavior for all backend in design:

mysql PK and UNIQUE: https://github.com/django/django/blob/e7f0e9b7045e7bcbe1e8ad85aec486c01f570e43/django/db/backends/mysql/introspection.py#L203 - just get all indexes within PRIMARY and UNIQUE CONSTRAINTS

sqlite PK: https://github.com/django/django/blob/fe1b51f6ed3fe4b41739647993cf0e3eac738f6c/django/db/backends/sqlite3/introspection.py#L426 - just set unique=False for PK

any thoughts for this situation?